src/resolvers/Query.js

```diff
@@ -1,16 +1,6 @@
 import getUserId from '../utils/getUserId';
 
 const Query = {
-  users(parent, args, { prisma }, info) {
-    return prisma.user.findMany({
-      where: {
-        username: {
-          contains: args.query
-        }
-      }
-    });
-  },
-
```

Comment:
Since we've now abstracted the Users from Characters, I've decided that it's not necessary to have this query anymore. We don't want people to be able to search for users and their account information, but we could do this for characters profiles later.

Reply:
Yes.

Reply:
Lmao